mautic/mautic
# Add support for RSS feed from mautic for user notifications

Please answer the following questions. 

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Related user documentation PR URL | N/A
| Related developer documentation PR URL | N/A
| Issues addressed (#s or URLs) | N/A
| BC breaks? | N
| Deprecations? | N

#### Description:
This PR adds support to the notifications tray to receive and display notifications from Mautic to keep you up to date on the latest news regarding updates, upcoming features, and general information from Mautic.

#### Steps to test this PR:
1. Apply the PR
2. run `composer update` from the command line.
3. Load your mautic dashboard, and you'll see new updates in your notification tray.
